src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.Core.targets

```diff
@@ -0,0 +1,169 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Bindings.Core.targets
+
+This file contains the core MSBuild logic for Xamarin.Android bindings.
+
+It is shared between "legacy" binding projects and .NET 5 projects.
+
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="Xamarin.Android.Tasks.BindingsGenerator" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+  <UsingTask TaskName="Xamarin.Android.Tasks.RemoveDirFixed"    AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+
+  <PropertyGroup>
+    <UseShortGeneratorFileNames Condition=" '$(UseShortGeneratorFileNames)' == '' ">false</UseShortGeneratorFileNames>
+    <GeneratedOutputPath        Condition=" '$(GeneratedOutputPath)' == '' ">$(IntermediateOutputPath)generated\</GeneratedOutputPath>
+    <ApiOutputFile              Condition=" '$(ApiOutputFile)' == '' ">$(IntermediateOutputPath)api.xml</ApiOutputFile>
+    <_GeneratorStampFile>$(IntermediateOutputPath)generator.stamp</_GeneratorStampFile>
+  </PropertyGroup>
+
+  <!-- Get our Build Actions to show up in VS -->
+  <ItemGroup>
+    <AvailableItemName Include="EmbeddedJar" />
+    <AvailableItemName Include="EmbeddedNativeLibrary" />
+    <AvailableItemName Include="EmbeddedReferenceJar" />
+    <AvailableItemName Include="InputJar" />
+    <AvailableItemName Include="ReferenceJar" />
+    <AvailableItemName Include="TransformFile" />
+    <AvailableItemName Include="LibraryProjectProperties" />
+    <AvailableItemName Include="LibraryProjectZip" />
+    <AvailableItemName Include="JavaDocIndex" />
+    <AvailableItemName Include="JavaDocJar" />
+    <AvailableItemName Include="JavaSourceJar" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <ExportJarToXmlDependsOnTargets>
```

Comment:
`jar2xml` related *anything* should be in `Xamarin.Android.Bindings.JarToXml.targets`.  `jar2xml` properties/items/targets/etc. *should not* exist in .NET 5.

Reply:
The target is named `ExportJarToXml`, even for `class-parse`...